packages/mcpplugin/README.md

```diff
@@ -0,0 +1,39 @@
+# Microsoft Teams MCP Plugin
+
+<p>
+    <a href="https://pypi.org/project/microsoft-teams-mcpplugin/" target="_blank">
+        <img src="https://img.shields.io/pypi/v/microsoft-teams-mcpplugin" />
+    </a>
+    <a href="https://pypi.org/project/microsoft-teams-mcpplugin/" target="_blank">
+        <img src="https://img.shields.io/pypi/dw/microsoft-teams-mcpplugin" />
+    </a>
+</p>
+
+Model Context Protocol (MCP) integration for Microsoft Teams AI applications.
```

Comment:
This package actually allows apps to both be MCP servers (expose tools to clients), and MCP clients (use mcp servers in their ai applications). You should show an example of each.